apps/meteor/client/components/UserInfo/UserInfoABACAttributes.tsx

```diff
@@ -0,0 +1,22 @@
+import { Box } from '@rocket.chat/fuselage';
+import type { ComponentProps, ReactElement } from 'react';
+
+import UserInfoABACAttribute from './UserInfoABACAttribute';
+
+type UserCardABACAttributesProps = {
+	abacAttributes: string[];
+} & ComponentProps<typeof Box>;
+
+const UserInfoABACAttributes = ({ abacAttributes, ...props }: UserCardABACAttributesProps): ReactElement => {
+	return (
```

Comment:
why all these props? `UserInfoABACAttribute` is a `Tag` which has the proper variant which handles color and fonts, please check this deeper, I'm quite sure its unnecessary, if so lets remove them